test/benchmark/e2e/http/main.go

```diff
@@ -156,28 +152,29 @@ func benchmarkReceiverSender(cases []e2e.BenchmarkCase, requestFactory func([]by
 	return results
 }
 
-type benchDelivery struct {
-	fn transport.DeliveryFunc
-}
-
-func (b *benchDelivery) Delivery(ctx context.Context, e event.Event) (*event.Event, transport.Result) {
-	return b.fn(ctx, e)
-}
-
-func dispatchReceiver(clients []cloudevents.Client, outputSenders int) transport.Delivery {
-	return &benchDelivery{fn: func(ctx context.Context, e cloudevents.Event) (*cloudevents.Event, error) {
-		var wg sync.WaitGroup
-		for i := 0; i < outputSenders; i++ {
-			wg.Add(1)
-			go func(client cloudevents.Client) {
-				_ = client.Send(ctx, e)
-				wg.Done()
-			}(clients[i])
-		}
-		wg.Wait()
-		return nil, nil
-	}}
-}
+//
```

Comment:
I'll take care of it when this PR is merged

Reply:
Thanks. I tired but it is complex code